tools/Makefile.unix

```diff
@@ -25,9 +25,13 @@ include $(TOPDIR)/Make.defs
 
 GIT_DIR = $(if $(wildcard $(TOPDIR)$(DELIM).git),y,)
 
+ifeq ($(GIT_DIR),y)
+GIT_PRESENT = `git rev-parse --git-dir 2> /dev/null`
+endif
+
```

Comment:
Doesn't Makefile.win need this change as well?

Reply:
I have the same question before, but Makefile.win don't generate version info from git history like Makefile.unix.

Reply:
I thought the same thing, but they diverged at some point.